### ISSUE TITLE
npm won't connect to Nuget to download "wildcard" package

### ISSUE THREAD

**Issue by nagi-desuuu** [cadon0/gif-to-the-beat#45]
```
npm ERR! code E401
npm ERR! 401 Unauthorized - GET https://nuget.lic.co.nz/npm/NPM/wildcard/-/wildcard-2.0.0.tgz
```

[2024-03-14T17_49_52_105Z-debug-0.log](https://github.com/cadon0/gif-to-the-beat/files/14606188/2024-03-14T17_49_52_105Z-debug-0.log)

**Comment by cadon0** [comment]
Hi, that problem was fixed (#42) but I'm not sure if I made a new download available. If you wanted to [view this file](https://github.com/cadon0/gif-to-the-beat/blob/master/package-lock.json) and select "Download raw file", then you can replace that file in the folder and it should be ok again
![image](https://github.com/cadon0/gif-to-the-beat/assets/25311843/994b1e68-728f-41fb-b6f9-b2876cd0a4ff)

**Comment by nagi-desuuu** [comment]
It does work now. That specific file never made it to a release.
